src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ContractTransaction, ethers } from "ethers";
+import { BigNumber, ContractTransaction, ethers } from "ethers";
```

Comment:
@JamesEarle I recommend you to use subpackages instead of `ethers` full packages, this will prevent package dependency issues in the frontend side.
i.e. `import { BigNumber } from "@ethersprojects/bignumber";`